src/config/NetworkConfig.ts

```diff
@@ -69,12 +69,12 @@ export const defaultTestnetNetworkConfig: NetworkConfig = {
         maxCosignedAccountsPerAccount: 25,
         maxMessageSize: 1024,
         maxMosaicAtomicUnits: 8999999999000000,
-        currencyMosaicId: '3A8416DB2D53B6C8',
-        harvestingMosaicId: '3A8416DB2D53B6C8',
+        currencyMosaicId: '72C0212E67A08BCE',
+        harvestingMosaicId: '72C0212E67A08BCE',
         defaultDynamicFeeMultiplier: 100,
         epochAdjustment: 1637848847,
```

Comment:
should the epochAdjustment change, it looks different from the network.

Reply:
updated

Reply:
good point @AnthonyLaw :+1: